hydra-main/src/main/java/com/addthis/hydra/job/alert/JobAlertManagerImpl.java

```diff
@@ -107,7 +140,169 @@ public String getAlert(String alertId) {
         return jobAlertRunner.getAlert(alertId);
     }
 
-    public Set<AbstractJobAlert> getAlertsForJob(String jobId) {
-        return jobAlertRunner.getAlertsForJob(jobId);
+    /**
+     * Create or update auto generated alerts on the job
+     *
+     * @param job         the job being updated
+     * @param basicAlerts true if we want basic alerts
+     * @param basicPages  true if we want basic pages
+     */
+    @Override public void updateBasicAlerts(final IJob job, final boolean basicAlerts, final boolean basicPages) {
+        @Nullable final Group group = this.groupManager.getGroup(job.getGroup());
+        // don't try to create alerts if config is not set up for the user/group
+        if (group == null) {
+            log.info("group was null");
```

Comment:
Use log.warn, and include more info in the message such as job id and group name. Ideally the user gets some kind of feedback for this, but that may be a lot more code than we want to change now.